packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-drilldown.vue

```diff
@@ -205,7 +223,14 @@ const knobs = ref<BasicKnobs>({
 	}
 });
 
+const newBlankInterventionPolicy = ref({
+	name: '',
+	modelId: '',
+	interventions: []
+});
```

Comment:
Why not using the [blankIntervention](https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/services/intervention-policy.ts#L5-L11)?